### PR TITLE
Show due date main page

### PR DESCRIPTION
**Pull request by te-sa**
Showing the due date in a TextView under the Checkbox now, default is "Due date: none"

### CODE REVIEW

File: app/src/main/res/layout/todo_item.xml [hunk 1]
```diff
@@ -4,35 +4,41 @@
...
-            android:layout_weight="1"
-            android:contentDescription="@string/button_to_edit_to_do"
-            app:srcCompat="@drawable/dots" />
-
-    </LinearLayout>
+    <TextView
+        android:id="@+id/dateView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
```
Comment (justino599) [inline app/src/main/res/layout/todo_item.xml:39]: android:layout_marginStart="62dp"

File: app/src/main/res/layout/todo_item.xml [hunk 2]
```diff
@@ -4,35 +4,41 @@
...
 
-    <LinearLayout
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:orientation="horizontal">
+    <CheckBox
+        android:id="@+id/checkBox"
+        android:layout_width="wrap_content"
+        android:layout_height="32dp"
+        android:layout_marginStart="20dp"
```
Comment (justino599) [inline app/src/main/res/layout/todo_item.xml:11]: android:layout_marginTop="8dp"

File: app/src/main/java/com/example/todolist/ToDoAdapter.java [hunk 3]
```diff
@@ -43,7 +46,13 @@ public ToDoAdapter.MyViewHolder onCreateViewHolder(@NonNull ViewGroup parent, in
     @Override
     public void onBindViewHolder(@NonNull ToDoAdapter.MyViewHolder holder, int position) {
         String toDoText = toDoList.get(position).getText();
+        Date toDoDate = toDoList.get(position).getDate();
+
+        String pattern = "MM-dd-yyyy";
```
Comment (justino599) [inline app/src/main/java/com/example/todolist/ToDoAdapter.java:51]: String pattern = "EEE, MMM dd";

**Review by justino599 — CHANGES_REQUESTED** [review]
I suggested a couple of layout changes, up to you if you think it looks better with them

**Comment by te-sa** [comment]
I implemented the layout changes you suggested, I think they definitely improved things

**Review by justino599 — APPROVED** [review]

**Pull request merged**